src/utils/graphQLClient.tsx

```diff
@@ -1,23 +1,28 @@
+import { useAddonState } from "@storybook/manager-api";
 import React from "react";
 import { Client, fetchExchange, Provider } from "urql";
 import { v4 as uuid } from "uuid";
 
 import { ACCESS_TOKEN_KEY, ADDON_ID, CHROMATIC_API_URL } from "../constants";
-import { useSharedState } from "./useSharedState";
 
 export { Provider };
 
 let currentToken: string = localStorage.getItem(ACCESS_TOKEN_KEY);
 const accessTokenSharedStateKey = `${ADDON_ID}/accessToken`;
 
 export const useAccessToken = () => {
-  const [token, setToken] = useSharedState<string>(accessTokenSharedStateKey, currentToken);
+  const [{ token }, setToken] = useAddonState<{ token: string | null }>(accessTokenSharedStateKey, {
+    token: currentToken,
+  });
```

Comment:
```suggestion
  // We use an object rather than a straight boolean here due to https://github.com/storybookjs/storybook/pull/23991
  const [{ token }, setToken] = useAddonState<{ token: string | null }>(accessTokenSharedStateKey, {
    token: currentToken,
  });
```